templates/registration/login.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+  <form method="post" action="{% url 'login' %}">
+    {% csrf_token %}
```

Comment:
The `{% csrf_token %}` tag is used to include a CSRF token in the form, which is crucial for preventing cross-site request forgery attacks.